.github/workflows/_tauri.yml

```diff
@@ -87,6 +87,8 @@ jobs:
       - name: Build release exe and MSI / deb
         # Signs the exe before bundling it into the MSI
         run: pnpm build
+      - name: Debug
+        run: git diff
```

Comment:
I'd like to leave this in since it happened twice and seeing `-modified` in the About window is pretty scary.

In fact, can we abort in CI if we detect that the build modified any files?

Reply:
> In fact, can we abort in CI if we detect that the build modified any files?

We should!

```suggestion
      - name: Debug
        run: git diff --exit-code
```